firmware/applications/SignBuddy/board.h

```diff
@@ -46,3 +47,35 @@
 #endif // defined(BOARD_DISCO)
 
 #endif // BOARD_H
+
+/* UART */
+#if defined(BOARD_DISCO)
+#define UART                          USART2
+#define UART_IRQn                     USART2_IRQn
+#define UART_IRQHandler               USART2_IRQHandler
+
+#define UART_RX_PIN                   LL_GPIO_PIN_3
+#define UART_TX_PIN                   LL_GPIO_PIN_2
+#define UART_GPIO_PORT                GPIOA
```

Comment:
where are these pins from?

Reply:
don't we need these pins in uart.c to configure under hw_init()?

Reply:
yes, but I don't think they are the correct pins

Reply:
confirmed they are correct